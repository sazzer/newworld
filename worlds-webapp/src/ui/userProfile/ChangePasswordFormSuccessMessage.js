// @flow

import React from 'react';
import {Message} from 'semantic-ui-react';
import {Interpolate} from 'react-i18next';

/**
 * The message to display for the user profile save result
 */
export default function ChangePasswordFormSuccessMessage() {
    return (
        <Message success>
            <Interpolate i18nKey='userProfile.password.messages.success' />
        </Message>
    )
}
