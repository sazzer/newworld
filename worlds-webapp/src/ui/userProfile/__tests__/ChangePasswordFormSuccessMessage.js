// @flow

import React from 'react';
import {shallow} from 'enzyme';
import ChangePasswordFormSuccessMessage from '../ChangePasswordFormSuccessMessage';

/** Create the component to test */
function setup() {
    return {
        element: shallow(<ChangePasswordFormSuccessMessage />)
    };
}

describe('ChangePasswordFormSuccessMessage', () => {
    it('Renders correctly', () => {
        const { element } = setup();

        expect(element).toMatchSnapshot();
    });
});
