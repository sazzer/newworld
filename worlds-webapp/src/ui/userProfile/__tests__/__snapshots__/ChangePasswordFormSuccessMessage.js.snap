// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangePasswordFormSuccessMessage Renders correctly 1`] = `
<Message
  success={true}
>
  <Interpolate
    className=""
    i18nKey="userProfile.password.messages.success"
  />
</Message>
`;
